c/httpserver.c

```diff
@@ -209,6 +209,14 @@ static char crlf[] ={ 0x0d, 0x0a};
 #define DEFAULT_UMASK 0022
 #endif
 
+//No need to fill logs with the same message, warn based on a desired frequency
+#ifdef __ZOWE_OS_ZOS
```

Comment:
Should this be `#if defined(__ZOWE_OS_ZOS) || defined(USE_RS_SSL)` instead?

Reply:
Yes, based on the code, we need to use these `#define`s because we won't be able to compile if `__ZOWE_OS_ZOS` is not defined.